src/components/BookSections/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { useBooksContext } from "../../store/books/BooksContext";
+import BookCollection from "../BookCollection";
+import Section from "../Section";
+
+const BookSections = () => {
+  const { allBooksCount } = useBooksContext();
+
+  return (
+    <Section title="Catalog" numberOfBooks={allBooksCount}>
```

Comment:
I wonder, why do you need a new API which returns the total number of books? Wouldn't it be enough to just do `books.length`?

Reply:
For now we render all books inside section, but later there will be "show more" button, and current number of displayed books won't be the same as number of all available books.